Makefile

```diff
@@ -9,12 +9,19 @@ integ: test
 fuzz:
 	go test -timeout=300s ./fuzzy
 	
-deps:
+deps: dep-linter
 	go get -t -d -v ./...
 	echo $(DEPS) | xargs -n1 go get -d
 
+lint:
+	gofmt -s -w .
+	golangci-lint run -c .golangci-lint.yml
+
+dep-linter:
+	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin latest
```

Comment:
This line still doesnt work. TIL In Make `$()` evaluates a variable instead of running a subshell. So its looking for the variable named `go env GOPATH` which returns the empty string.